auditbeat/docs/fields.asciidoc

```diff
@@ -2914,8 +2914,15 @@ type: object
 [[exported-fields-ecs]]
 == ECS fields
 
-ECS Fields.
 
+This section defines Elastic Common Schema (ECS) fields—a common set of fields
```

Comment:
Do we want to call out that ECS is an open source specification? Otherwise, it LGMT!